definitions/v3/blutopia-api.yml

```diff
@@ -54,10 +54,24 @@ search:
   paths:
     # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
     # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
-    - path: "/api/torrents/filter?api_token={{ .Config.apikey }}&name={{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}&sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    - path: "/api/torrents/filter?"
       response:
         type: json
         attribute: attributes
+  inputs:
+  # if we have an id based search, add Season and Episode as query in name for UNIT3D < v6.  Else pass S/E Params for UNIT3D >= v6
+    $raw: "{{ range .Categories }}categories[]={{.}}&{{ end }}{{ if .Config.freeleech }}freeleech=1&{{ else }}{{ end }}"
```

Comment:
Getting errors here

System.Collections.Generic.KeyNotFoundException: The given key '.}}&{{ end' was not present in the dictionary.

Reply:
@ilike2burnthing does raw have issues with range for categories?

Reply:
Freeleech needs to moved down or the range needs to be wrapped in an if. Parser doesn't like the range and then if

Reply:
Just the range works fine `$raw: "{{ range .Categories }}categories[]={{.}}&{{end}}"` 

so you need the freeleech if to wrap the range, or you need to move out freeleech

Reply:
Turns out `{{.}}&{{end}}` wasn't a typo, it is necessary to avoid that error (tested with torrentdb and torrenting).

Reply:
Errors out:

```yml
  $raw: "{{ range .Categories }}categories[]={{.}}&{{ end }}{{ if .Config.freeleech }}freeleech=1&{{ else }}{{ end }}"
```

Working:

```yml
  $raw: "{{ range .Categories }}categories[]={{.}}&{{end}}{{ if .Config.freeleech }}freeleech=1&{{ else }}{{ end }}"
```